### ISSUE TITLE
Smartblock [FEATURE]

### ISSUE THREAD

**Issue by DGWAsureX** [CCBlueX/LiquidBounce1.8-Issues#3655]
I would like a smartblock for killaura.

**Comment by apexCode1337** [comment]
Yeah you could add many checks

**Comment by CzechHek** [comment]
CubeCore

**Comment by apexCode1337** [comment]
Yeah of course but there are many checks missing because you don't have the ability to make them in scripts.

**Comment by CzechHek** [comment]
> Yes of course but there are many checks missing because you don't have the ability to make them in scripts.

Not that I don't have the ability in ScriptAPI, there was no point.
